docs/running-a-node/troubleshooting.md

```diff
@@ -10,6 +10,45 @@ The default location of standard node log files for various systems:
 
 ---
 
+## Troubleshooting UPnP
+
+### Ensure UPnP is enabled
+UPnP will be enabled unless the external port is set in either the config
+```bash
+[node]
+...
+# The external address of this node (NAT). If not set, the node will request this information via UPnP.
+# type:string,ip
+external_address = "::ffff:<some_public_ipv4>"
+```
+or via cli flag
+```bash
+--config node.external_address="::ffff:<some_public_ipv4>"
+```
+
+### Enable UPnP logging
+Appending this to your launch command will enable upnp logging.
+```bash
+--config node.logging.upnp_details=true
+```
+### Check Logs for responses
+Check the begginning of the logs for UPNP_* messages
+
+!!! warning "Standard message indcating no existing mapping"
```

Comment:
So when I get this message it's working as intended, and when the UPnP loops again this message will not be there as the mapping was created? If so, as a user this message reads like an error, and maybe we should either:

1. Make a big notice here that it is the intended behavior, or
2. In the node, log differently when the error code is 714 to make it look less like an error

I like 2 more, as the user will not need to check the documentation, but could go with either.

Also small typo, ind_i_cating